support-frontend/app/models/identity/responses/IdentityErrorResponse.scala

```diff
@@ -29,12 +29,14 @@ object IdentityErrorResponse {
   case class IdentityError(
       message: String,
       description: String,
+      context: String = "primaryEmailAddress",
   )
 
   object IdentityError {
     object InvalidEmailAddress {
```

Comment:
Are we categorising the bad email errors that come from Identity as `InvalidEmailAddress` errors? When I test the form now I don't see this `message` "Registration Error" or the `description` "Please sign up using an email address from a different provider", I see a generic error in the client...

<img width="431" alt="Screenshot 2022-11-25 at 16 53 45" src="https://user-images.githubusercontent.com/1590704/204029514-f868328a-9797-4d26-a34e-2b03b8c2d753.png">

Reply:
I think the `InvalidEmailAddress` error was originally set-up to handle email addresses that were on Identity's disallowed email provider list, rather than email errors originating from OKTA.